src/plugins/tabs/components/tabs.vue

```diff
@@ -49,11 +52,12 @@
     <div
         v-for="tab in tabsList"
         :key="tab.keyString"
+        :style="getTabStyles(tab)"
         class="c-tabs-view__object-holder"
         :class="{'c-tabs-view__object-holder--hidden': !isCurrent(tab)}"
     >
         <object-view
-            v-if="isTabLoaded(tab)"
+            v-if="shouldLoadTab(tab)"
```

Comment:
We should avoid a method call here as well. Couple of options:

- Tabs have a property on themselves that determines if they are active
- A computed property that returns a map of active tabs keyed on tab.keyString.